### ISSUE TITLE
TypeError: is instance()

### ISSUE THREAD

**Issue by RomanProkazov** [Hosea174/spotify-downloader#3]
After entering the URL, have this error:
  File "/usr/local/lib/python3.11/site-packages/spotipy/oauth2.py", line 156, in __del__
TypeError: isinstance() arg 2 must be a type, a tuple of types, or a union

**Comment by Hosea174** [comment]
I abandoned this project like a year ago, it was just a project for this course I was taking... I didn't expect this many people to be interested in checking it out. I've been busy with college and other projects, but I'll definitely get back to this and actually create a UI for it as well. Thanks for sharing the issue, feel free to let me know if you figure out the problem.

**Comment by RomanProkazov** [comment]
Yeah, Actualy I wanted to download all songs from my spotify playlists to make dj sets. I think Will do sc50 course myself then, nice job anyway💪

**Comment by Hosea174** [comment]
Thanks brother! And you should definitely take cs50, good luck!

**Comment by kh4n9373** [comment]
So where is the solution to this issue ?
